pkg/graph/schema.graphql

```diff
@@ -2161,6 +2172,10 @@ type Mutation {
   ): CreateSystemIntakeActionExtendLifecycleIdPayload @hasRole(role: EASI_GOVTEAM)
   createSystemIntakeNote(input: CreateSystemIntakeNoteInput!): SystemIntakeNote
     @hasRole(role: EASI_GOVTEAM)
+  updateSystemIntakeNote(input: UpdateSystemIntakeNoteInput!): SystemIntakeNote!
+    @hasRole(role: EASI_GOVTEAM)
+  setSystemIntakeNoteArchived(id: UUID!, isArchived: Boolean!): SystemIntakeNote!
```

Comment:
Do you think it would be easier to just have one mutation to update a system intake note, and allow that update mutation to edit the note including the `isArchived` field? That way you could just add `isArchived` to `UpdateSystemIntakeNoteInput` and get rid of the `setSystemIntakeNoteArchived` mutation

Reply:
I did think about that - but i ended up just using the way that the TRB admin notes function (which use separate archive and update mutations) as a baseline. I am fine with switching to a singular update call for both and do agree it is probably cleaner.

Reply:
@ClayBenson94 - thoughts? I already made the change that Jeremy is proposing and it works so I can push it now but wanted to get your opinion on which path you prefer.

Reply:
****

Reply:
Per our review convo, let's make it one!